pyinfra_cli/prints.py

```diff
@@ -143,12 +143,15 @@ def print_facts(facts):
 
 def print_support_info():
     click.echo('''
-    If you're having issues with pyinfra or wish to make feature requests, please
-    check out the GitHub issues at https://github.com/Fizzadar/pyinfra/issues.
+    If you are having issues with pyinfra or wish to make feature requests, please
+    check out the GitHub issues at https://github.com/Fizzadar/pyinfra/issues .
```

Comment:
Why the space before the full stop, out of interest?

Reply:
Didn’t want any confusion on url. 

Ok if you want to remove it.